bigquery_etl/feature_usage/templating.yaml

```diff
@@ -0,0 +1,1085 @@
+        vetted: true
+        min_version: 50
+        type: INT64
+      - name: scalar_parent_browser_engagement_unique_domains_count_mean
+        sql: scalar_parent_browser_engagement_unique_domains_count_mean
+        vetted: true
+        min_version: 50
+        type: INT64
+      - name: scalar_parent_browser_engagement_max_concurrent_tab_count_max
+        sql: scalar_parent_browser_engagement_max_concurrent_tab_count_max
+        vetted: true
+        min_version: 50
+        type: INT64
+      - name: scalar_parent_browser_engagement_max_concurrent_window_count_max
+        sql: scalar_parent_browser_engagement_max_concurrent_window_count_max
+        vetted: true
+        min_version: 50
+        type: INT64
+      - name: search_count_abouthome
+        sql: search_count_abouthome
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_count_all
+        sql: search_count_all
+        vetted: true
+        min_version: 43
+        type: INT64
+      - name: search_count_contextmenu
+        sql: search_count_contextmenu
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_count_newtab
+        sql: search_count_newtab
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_count_organic
+        sql: search_count_organic
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_count_searchbar
+        sql: search_count_searchbar
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_count_system
+        sql: search_count_system
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_count_tagged_follow_on
+        sql: search_count_tagged_follow_on
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_count_tagged_sap
+        sql: search_count_tagged_sap
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_count_urlbar
+        sql: search_count_urlbar
+        vetted: true
+        min_version: 86
+        type: INT64
+      - name: search_with_ads_count_all
+        sql: search_with_ads_count_all
+        vetted: true
+        min_version: 64
+        type: INT64
+  ### Main ###
+  - name: main
+    ref: telemetry.main_1pct
+    group_by: ["submission_date", "client_id"]
+    filters:
+      - sql: DATE(submission_timestamp) = @submission_date
+      - sql: sample_id = 0
+      - sql: normalized_channel = 'release'
+    measures:
+      - name: client_id
+        sql: client_id
+      - name: subsample_id
+        sql: MOD(ABS(FARM_FINGERPRINT(client_id)), 100)
+      - name: submission_date
+        sql: DATE(submission_timestamp)
+      - name: is_headless
+        sql: LOGICAL_OR(COALESCE(environment.system.gfx.headless, FALSE))
+        vetted: true
+        min_version: 72
+        type: BOOL
```

Comment:
This is the file for configuring the metrics that will be used for generating the query. This config defines `sources` which are tables to query from and each source has a set of `measures` which are the feature usage metrics we are interested in. For each measure the `sql` expression needs to be defined, and optionally a `min_version` a `type` and whether the metric is `vetted`.